tests/test_http_server.py

```diff
@@ -0,0 +1,67 @@
+import unittest
+
+import pytest
+import requests
+
+
+@pytest.mark.dockertest
+@pytest.mark.httpservertest
+def test_password_cluster_is_up(local_docker_cluster_passwd):
+    cluster = local_docker_cluster_passwd
+    cluster.check_server()
+    assert cluster.is_up()  # Should be true for a Cluster object
+
+
+@pytest.mark.dockertest
+@pytest.mark.httpservertest
+def test_public_key_cluster_is_up(local_docker_cluster_public_key):
+    cluster = local_docker_cluster_public_key
+    cluster.check_server()
+    assert cluster.is_up()  # Should be true for a Cluster object
+
+
+@pytest.mark.dockertest
```

Comment:
I've created a new file called [`test_telemetry.py`](https://github.com/run-house/runhouse/blob/d1993e3e8fce782fae5f1001bb7b5b5f32c07870/tests/test_telemetry.py#L9). This PR is already merged into main.
In the PR, I've moved the following method into `test_telemetry.py`:
`def test_public_key_cluster_has_telemetry(local_docker_cluster_telemetry_public_key):`

Should we move `def test_public_key_cluster_has_telemetry_with_tls(
    local_docker_cluster_telemetry_public_key_with_tls,` there too?

Also, let's annotate both tests with
`
@pytest.mark.dockertest
@pytest.mark.telemetrytest
`
Thanks!